templates/tail-file.conf.erb

```diff
@@ -6,7 +6,9 @@
 <% @matches.each do |match| -%>
   <Match>
    Regex "<%= match['regex'] %>"
+<% if match['excluderegex'] -%>
    ExcludeRegex "<%= match['excluderegex'] %>"
+<% end -%>   
```

Comment:
which results in extra 'blank' lines in the outputted file...
https://travis-ci.org/voxpupuli/puppet-collectd/jobs/132898797#L1607